.gitattributes

```diff
@@ -0,0 +1 @@
+* text=auto
```

Comment:
Why did you remove the _.placeholder_ file? (Can't comment in that one directly.)

Reply:
See https://github.com/Lombiq/Orchard-UIKit/pull/13#discussion_r856669196